srcpkgs/gnome/template

```diff
@@ -11,69 +11,69 @@ homepage="https://www.gnome.org"
 depends="
  gnome-core>=${version}
  baobab>=${version}
- cheese>=${version}
+ cheese>=41.1
  eog>=${version}
  evince>=${version}
- file-roller>=3.40.0
- gedit>=40.1
+ file-roller>=3.${version}
  gnome-calculator>=${version}
  gnome-calendar>=${version}
  gnome-characters>=${version}
- gnome-clocks>=40.0
+ gnome-clocks>=${version}
  gnome-contacts>=${version}
  gnome-disk-utility>=${version}
  gnome-font-viewer>=${version}
  gnome-maps>=${version}
  gnome-music>=${version}
  gnome-remote-desktop>=${version}
- gnome-screenshot>=40.0
```

Comment:
I don't think removing gnome-screenshot from gnome is a good decision.

Reply:
Alright, I just didn't think it was necessary anymore since it was replaced by the shell UI.

Reply:
I forgot about that, thanks for pointing that out. I think we can remove it then.

Reply:
Okay.